app/helpers/settings_schedule_helper.rb

```diff
@@ -0,0 +1,118 @@
+      settings_schedule_filter_details(schedule, exp_table),
+    ]
+    summary.push(settings_schedule_object_details(schedule)) if automation_request_check
+    summary.push(settings_schedule_object_attribute(object_class, object_name)) if automation_request_check
+    summary.push(settings_schedule_attribute_value_pairs(schedule)) if automation_request_check && schedule.filter[:ui][:ui_attrs].present?
+    safe_join(summary)
+  end
+
+  def settings_schedule_basic_info(schedule)
+    data = {
+      :title => _('Basic Information'),
+      :mode  => "settings_schedule_basic_info",
+    }
+
+    rows = [
+      row_data(_('Description'), schedule.description),
+      row_data(_('Active'), schedule.enabled),
+    ]
+
+    if schedule.sched_action[:method] && schedule.sched_action[:method] == "check_compliance"
+      rows.push(row_data(_('Action'), "#{ui_lookup(:model => schedule.resource_type)} #{_('Compliance Check')}"))
+    elsif schedule.sched_action[:method] && schedule.sched_action[:method] == "automation_request"
+      rows.push(row_data(_('Action'), _('Automate Task')))
+    else
+      rows.push(row_data(_('Action'), "#{ui_lookup(:model => schedule.resource_type)} #{_('Analysis')}"))
+    end
+
+    data[:rows] = rows
+    miq_structured_list(data)
+  end
+
+  def settings_schedule_filter_details(schedule, exp_table)
+    data = {
+      :title => _('Filter'),
+      :mode  => "settings_schedule_filter_details",
+    }
+
+    rows = []
+    unless schedule.sched_action[:method] == "automation_request"
+      if schedule.miq_search
+        search = schedule.miq_search
+        description = "Filter: #{search.description}"
+        filter_data = search.search_type == "user" ? "My #{description}" : "Global #{description}"
+        rows.push({:cells => [{:value => filter_data}]})
+      elsif schedule.filter.kind_of?(MiqExpression)
+        operators = ["AND", "OR", "(", ")"]
+        exp_table.each do |token|
+          if operators.exclude?([token].flatten.first)
+            rows.push({:cells => [{:value => [token].flatten.first}]})
+          else
+            rows.push({:cells => [{:value => [token].flatten.first, :style => 'color_black'}]})
+          end
+        end
+      else
+        rows.push({:cells => [{:value => MiqExpression.to_human(schedule.expression)}]})
+      end
+    end
+
+    data[:rows] = rows
+    miq_structured_list(data)
+  end
+
+  def settings_schedule_object_details(schedule)
+    rows = [
+      row_data(_('System/Process'), schedule.filter[:uri_parts][:instance]),
+      row_data(_('Message'), schedule.filter[:uri_parts][:message]),
+    ]
+
+    if schedule.next_run_on.present?
+      rows.push(row_data(_('Request'), schedule.filter[:uri_parts][:message]))
+    else
+      rows.push(row_data(_('Request'), ''))
+    end
+
+    data = {
+      :title => _('Object Details'),
+      :mode  => "settings_schedule_object_details",
+      :rows  => rows
+    }
+    miq_structured_list(data)
```

Comment:
Up to you (personal preference) but if you aren't using the `data` local variable you can just pass the hash to `miq_structured_list`
```suggestion
    miq_structured_list(
      :title => _('Object Details'),
      :mode  => "settings_schedule_object_details",
      :rows  => rows
    )
```

Reply:
fixed